Obsidian.API/Noise/TerrainSelect.cs

```diff
@@ -4,20 +4,21 @@ namespace Obsidian.API.Noise;
 
 public class TerrainSelect : SharpNoise.Modules.Blend
 {
-    internal Dictionary<Biomes, Module> TerrainModules { get; set; } = new();
+    private static readonly Constant zero = new() { ConstantValue = 0 };
 
     public Module BiomeSelector { get; set; }
+    internal Dictionary<Biomes, Module> TerrainModules { get; set; } = new();
 
     public TerrainSelect(Module biomeSelector)
     {
-        Source1 = new Constant { ConstantValue = 0 };
+        Source1 = zero;
```

Comment:
This is a huge problem I need some help solving someday. Basically, the only way I could figure out how to merge biome's terrain together was to have them meet at sea level... which works great for overworld biomes, but looks awful when there are 2 adjacent ocean biomes.

Someday...